includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -267,6 +268,46 @@ public function admin_options() {
 		parent::admin_options();
 	}
 
+	/**
+	 * Generates the configuration values, needed for payment fields.
+	 *
+	 * Isolated as a separate method in order to be avaiable both
+	 * during the classic checkout, as well as the checkout block.
+	 *
+	 * @return array
+	 */
+	public function get_payment_fields_js_config() {
+		return [
+			'publishableKey'         => $this->account->get_publishable_key( $this->is_in_test_mode() ),
+			'accountId'              => $this->account->get_stripe_account_id(),
+			'ajaxUrl'                => admin_url( 'admin-ajax.php' ),
+			'updateOrderStatusNonce' => wp_create_nonce( 'wcpay_update_order_status_nonce' ),
+			'createSetupIntentNonce' => wp_create_nonce( 'wcpay_create_setup_intent_nonce' ),
+			'genericErrorMessage'    => __( 'There was a problem processing the payment. Please check your email and refresh the page to try again.', 'woocommerce-payments' ),
```

Comment:
By "check your email" do you mean their email inbox or do you mean the address on the screen. (I think you mean the former.) Please add "inbox" or "address" as appropriate after email.

Reply:
I added `inbox` in https://github.com/Automattic/woocommerce-payments/pull/723/commits/317b4b013306580b466277a9c417edafa49c10ab